Rakefile

```diff
@@ -12,7 +12,7 @@ end
 
 task :test => [:check_test]
 
-task :generate_host_config => [:check_acceptance]do |t, args|
+task :generate_host_config do |t, args| #=> [:check_acceptance]do |t, args|
 
   target = ENV["TEST_TARGET"] || 'centos7-64'
   generate = "bundle exec beaker-hostgenerator"
```

Comment:
until we have add_arg() or whatever, we could use a flag and ENV for #{target} below.  and command.  oh, and we shouldn't need to bundle exec here